apps/frontend/src/renderer/components/forgejo-issues/components/InvestigationDialog.tsx

```
@@ -0,0 +1,256 @@
+                    {selectedCommentIds.length === comments.length ? t('investigation.deselectAll') : t('investigation.selectAll')}
+                  </Button>
+                </div>
+                <ScrollArea
+                  className="flex min-h-0 border rounded-md"
+                  viewportClassName="h-auto"
+                >
+                  <div className="p-2 space-y-2">
+                    {comments.map((comment) => (
+                      <div
+                        key={comment.id}
+                        role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)